src/engines/html5/media-source/adapters/native-adapter.js

```diff
@@ -460,6 +466,7 @@ export default class NativeAdapter extends BaseMediaSourceAdapter {
         this._trigger(Html5EventType.WAITING);
       }
     }
+    this._handleVideoTracksChange();
```

Comment:
Do we must check it on every time update instead of tracks changed?

Reply:
SmartTV doesn't have tracks change